gracedemo/demo.go

```diff
@@ -9,7 +9,7 @@ import (
 	"os"
 	"time"
 
-	"github.com/facebookgo/grace/gracehttp"
+	"github.com/absolute8511/grace/gracehttp"
```

Comment:
This was probably flipped for testing, but we should undo this before merging here.